installation/cmd/run.sh

```diff
@@ -5,13 +5,19 @@ set -o errexit
 ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../..
 defaultRelease=$(<"${ROOT_PATH}"/installation/resources/KYMA_VERSION)
 KYMA_RELEASE=${1:-$defaultRelease}
-INSTALLER_CR_PATH="${ROOT_PATH}"/installation/resources/installer-cr-kyma-diet.yaml
+INSTALLER_CR_PATH="${ROOT_PATH}"/installation/resources/installer-cr-kyma-dependencies.yaml
 OVERRIDES_COMPASS_GATEWAY="${ROOT_PATH}"/installation/resources/installer-overrides-compass-gateway.yaml
 ISTIO_OVERRIDES="${ROOT_PATH}"/installation/resources/installer-overrides-istio.yaml
 API_GATEWAY_OVERRIDES="${ROOT_PATH}"/installation/resources/installer-overrides-api-gateway.yaml
 
+if [[ $KYMA_RELEASE != *PR-* ]] && [[ $KYMA_RELEASE != *master* ]]; then
```

Comment:
I think you got this backwards, in case of release we don't need to grep the image tag, as it's the same as the value in file. We need to do it in case of PR.

Reply:
- run script during run.sh: <- used by local development + compass-integration
    1. read KYMA_RELEASE content
    2. Get combo yaml:
        1. “PR-“ / “master” prefix -> look for directory in kyma-development-artifacts
        2. otherwise: try to download https://github.com/kyma-project/kyma/releases/download/{RELEASE}/kyma-installer-local.yaml
    3. get installer image tag
    4. Use this tag as installer tag for —source in CLI